### DISCUSSION TITLE
Is there a way to backup all customizations?

### DISCUSSION THREAD

**Discussion by one3711** [Flow-Launcher/Flow.Launcher#2948] [Q&A]
I have to uninstall the launcher for security reasons. Is there a way to save all the customisation including key shortcuts (within different plugins) as I am very sure I will download it again.

**Comment by jjw24** [comment]
Hi there, enable portable mode, move it to backup area. When enabling portable it is like uninstalling in a way, it removes the uninstaller entry and your UserData folder back into flow, allowing you to move the entire flow folder else where.

If you just want to back up flow settings, customisation, plugins and etc, simply copy the UserData folder else where. It contains everything you need to return back to the current state.

**Comment by jjw24** [comment]
What security issue did you run into with flow?

**Reply by one3711** [reply]
Not exactly a security issue but I've recently joined a uni and they've asked me to uninstall everything or factory reset (we're supposed to buy a new laptop but I already have one) and they will be checking each one's and install some things. I can install flow or any other software after this checking ends.
